cloud/service-metrics.md

```diff
@@ -0,0 +1,64 @@
+# Service metrics
+Timescale Cloud provides a Metrics dashboard for managing your services. You can
+see the Metrics dashboard in your Timescale Cloud account by navigating to the
+`Services` section, clicking the service you want to explore, and selecting the
+`Metrics` tab.
+
+You can view metrics for your services for any of these time ranges:
+*   Last hour, with one minute granularity
+*   Last 24 hours, with one minute granularity
+*   Last seven days, with one hour granularity
+*   Last 30 days, with one hour granularity
+
+To change the view, select the time range from the drop-down menu.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-metrics_lastmonth.png" alt="Timescale Cloud Metrics dashboard"/>
+
+Additionally, you can turn automatic metric refreshes on and off. When automatic
+metric refresh is on, the dashboard updates every thirty seconds.
+
+In some cases, gray vertical bars display on the metrics dashboard, like this:
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-metrics_graybar.png" alt="Timescale Cloud Metrics not collected"/>
+
+This indicates that metrics have not been collected for the period shown. It
+does not mean that your Timescale Cloud service was down.
+
+## Continuous storage monitoring
+Timescale Cloud continuously monitors the health and resource consumption of all
+database services. You can check your health data by navigating to the `metrics`
+tab in your service dashboard. These metrics are also monitored by the Timescale
+operations team.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-disk-metrics.png" alt="Timescale Cloud metrics dashboard"/>
+
+If your database exceeds a storage threshold of available resources, some
+automated actions are triggered, including notifications, and preventative
+actions.
+
+## Automated user alerting
```

Comment:
Should these be subsections of "Continuous storage monitoring" since they're mentioned as triggered actions from it?

Reply:
Ah, yes, I think you're right.

Reply:
```suggestion
### Automated user alerting
```